LayoutTests/transitions/flex-transitions.html

```diff
@@ -14,60 +15,69 @@
     background-color: green;
 }
 .flexbox div {
-    -webkit-transition-property: -webkit-flex;
-    -webkit-transition-duration: 1s;
-    -webkit-transition-timing-function: linear;
+    transition-property: flex;
+    transition-duration: 1s;
+    transition-timing-function: linear;
 }
 
 #flex-row div {
-    flex: 1 0 0px;
+  flex: 100 0 0px;
```

Comment:
Can you make indentation consistent?

Reply:
Ops! Messed-up. Just did all with 4 spaces for CSS style. :-)